.ebextensions/aws-autoscaling.config

```diff
@@ -1,6 +1,6 @@
 option_settings:
   aws:ec2:instances:
-    InstanceTypes: r5.large
+    InstanceTypes: m6i.2xlarge
```

Comment:
@oblodgett r5.large to m6i.2xlarge means going from 16 GiB and 2vCPU to 32 GiB and and 8vCPU. Would you want to upgrade one or the other, not both (and definitely not going from 2vCPU to 8 while we don't event see a bottleneck at vCPU level on the app server)?

Reply:
Im looking for both more memory and more cpu... there is something in this chain that is a bottle neck and right now its not easy to identify so I would like to open up the pipe to see the better performance. And then start dropping pieces of it to understand what is the effecting thing.

Reply:
Right now ES is on a 16 node cluster, which is massive compared to what it was on. The database and its IO have been doubled, and yet no change. So it has to be the app server, and my guess is that we might have several rounds of sizing to get the right instance type.

Reply:
Alright, if upsizing vCPU and RAM on the app server doesn't work, then you might want to look at trying to improve network and disk. `m6id.2xlarge` would be the equivalent disk optimised instance, `m6in.2xlarge` the network optimised one and `m6idn.2xlarge` both network and disk optimised. We may also need to check if we need to change any system configurations to make use of that improved network (not sure).